### PR TITLE
docs: adding guide for packaging devfile

### PR DESCRIPTION
**Pull request by mike-hoang**
## Which issue(s) does this PR fix

Fixes https://github.com/devfile/api/issues/1071

## PR acceptance criteria

- [ ] Unit Tests
- [ ] E2E Tests
- [x] Documentation
_Update the [sidebar](https://github.com/devfile/devfile-web/tree/main/apps/landing-page#configuring-navigation) if there is a new file added or an existing filename is changed_

### CODE REVIEW

File: libs/docs/src/docs/2.2.0/packaging-devfile.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{% callout title="Note!" %}
+Check out the [devfile schema](./devfile-schema) for supported components.
+{% /callout %}
+
+
+## Additional resources
+
+For more information about working with devfiles, go to [How to work with devfiles](./how-to-work-with-devfiles).
+
+For more information about outerloop, go to [Innerloop versus outerloop](./innerloop-vs-outerloop).
```
Comment (michael-valdron) [inline libs/docs/src/docs/2.2.0/packaging-devfile.md:43]: I feel we should using a bullet list for Additional resources, though I am open to a second opinion.
```suggestion
## Additional resources

- For more information about working with devfiles, go to [How to work with devfiles](./how-to-work-with-devfiles).
- For more information about outerloop, go to [Innerloop versus outerloop](./innerloop-vs-outerloop).
```

File: libs/docs/src/docs/2.2.1-alpha/packaging-devfile.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+{% callout title="Note!" %}
+Check out the [devfile schema](./devfile-schema) for supported components.
+{% /callout %}
+
+
+## Additional resources
+
+For more information about working with devfiles, go to [How to work with devfiles](./how-to-work-with-devfiles).
+
+For more information about outerloop, go to [Innerloop versus outerloop](./innerloop-vs-outerloop).
```
Comment (michael-valdron) [inline libs/docs/src/docs/2.2.1-alpha/packaging-devfile.md:43]: Same here.
```suggestion
## Additional resources

- For more information about working with devfiles, go to [How to work with devfiles](./how-to-work-with-devfiles).
- For more information about outerloop, go to [Innerloop versus outerloop](./innerloop-vs-outerloop).
```

**Review by michael-valdron — CHANGES_REQUESTED** [review]
Other than the Additional resources feedback, this lgtm.

File: libs/docs/src/docs/2.2.0/packaging-devfile.md (hunk 1 above)
Comment (mike-hoang) [reply libs/docs/src/docs/2.2.0/packaging-devfile.md:43]: No strong preference from me; we seem to use no bullets if it's a single item and bullets for multiple

Let's just stick to bullets going forward

**Review by michael-valdron — APPROVED** [review]
/lgtm

**Pull request merged**